spec/models/order_spec.rb

```diff
@@ -1,5 +1,49 @@
 require 'rails_helper'
 
 RSpec.describe Order, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:order){
```

Comment:
phân biệt let và let!

Reply:
let! sẽ tạo mới một đối tượng mỗi khi một test case được thực hiện (Ngầm định khởi tạo trong một khối do end trước khối do end của it) và 1 đối tượng được tạo bởi let! chỉ tồn tại trong khối do end của it. let sẽ chỉ tạo đối tượng nếu test case có yêu cầu gọi đến một đối tượng cụ thể